XenModel/Messages.resx

```diff
@@ -2166,6 +2166,9 @@ Deleting this bond will disrupt traffic through the secondary interface on the b
   <data name="CANNOT_ACTIVATE_VD_VM_HALTED" xml:space="preserve">
     <value>It is not possible to activate this virtual disk on VM '{0}' as the VM is not running.</value>
   </data>
+  <data name="CANNOT_ACTIVATE_VD_VM_NEEDS_IO_DRIVERS" xml:space="preserve">
+    <value>You must install the I/O drivers on VM '{0}' before you can activate this virtual disk for the VM.</value>

```

Comment:
"the I/O drivers" sounds OK, but in most other places we just refer to "I/O drivers", so let's drop the "the" here to make it consistent.